app/models/invenio_rdm_record_converter.rb

```diff
@@ -263,12 +264,18 @@ def rights(license_urls)
     license_urls.map do |license_url|
       license_data = @@license_data[license_url.to_sym]
 
-      {
-        "rights": license_data[:"name"],
-        "scheme": DEFAULT_RIGHTS_SCHEME,
-        "identifier": license_data[:"licenseId"],
-        "url": license_url
-      }
+      if license_data.present?
+        {
+          "rights": license_data[:"name"],
+          "scheme": DEFAULT_RIGHTS_SCHEME,
+          "identifier": license_data[:"licenseId"],
+          "url": license_url
+        }
+      elsif license_url == ALL_RIGHTS_RESERVED
+        {
+          "rights": ALL_RIGHTS_RESERVED
```

Comment:
How about using `ALL_RIGHTS_RESERVED` with `@@license_data`? Since we have that data, it would fill in the other fields(scheme, identifier, and url), right?